doc/level1.rst

```diff
@@ -682,12 +685,15 @@ this case contains one Autocrypt-Gossip header for each recipient,
 each of which:
 
 - MUST include an ``addr`` attribute that matches one of the
-  recipients in the ``To`` or ``Cc`` headers.
+  recipients in the ``To``, ``Cc``, or ``Reply-To`` headers.
```

Comment:
I think this should change "recipients" to "addresses", since the address in a `Reply-To` header is not a recipient.